nixos/modules/services/x11/desktop-managers/surf-display.nix

```diff
@@ -69,15 +69,15 @@ in {
       };
 
       screensaverSettings = mkOption {
-        type = types.string;
+        type = types.str;
```

Comment:
I think this could be `separatedString " "` since its simply passed on as [command line options to xset](https://code.it-zukunft-schule.de/cgit/surf-display/tree/bin/surf-display?id=972d6c4b7c4503dbb63fa6c92cdc24d1e32064a4#n182).